database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,10 +14,8 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
```

Comment:
yeah, when i made aurora it was just a part of the user table automatically, and I think I needed it b/c i had the roles pivot table since it worked as the main id. I don't think it's needed or used though